app/controllers/api/v1/admin/server_rooms_controller.rb

```diff
@@ -6,18 +6,22 @@ module Admin
       class ServerRoomsController < ApiController
         before_action :find_server_room, only: :destroy
 
+        # GET /api/v1/admin/server_rooms.json
         def index
-          rooms = Room.all.search(params[:search])
-
-          rooms.map! do |room|
-            {
-              friendly_id: room.friendly_id,
-              name: room.name,
-              owner: User.find(room.user_id).name
-            }
+          rooms = Room.includes(:user).search(params[:search])
+          active_rooms = BigBlueButtonApi.new.active_meetings
```

Comment:
Using the BigBlueButtonApi `active_meetings` method is good because it fetches all the active meetings in a single API call rather than making individual requests for each room, which would be much slower and inefficient.